src/pages/resources/data-structures-and-algorithms.mdx

```diff
@@ -3,61 +3,89 @@ layout: ../../layouts/MDLayout.astro
 title: Data Structures and Algorithms Resources
 description: A compilation of resources of learning data structures and algorithms!
 keywords: ["dsa", "cp", "competitive programming"]
-
-hidetoc: true
 ---
 
 This page is a compilation of *free* resources for people to use. The categories are taken from [neetcode](https://neetcode.io/practice) since we're currently (as of writing) following this study plan as a community.
 
 ## General Resources
 
+Here's a bunch of different resources to check out
```

Comment:
ratio

Reply:
ratio

Reply:
making a comment for discord webhook testing

Reply:
i am once again making a comment for discord webhook testing